libs/auth/src/cac/common_access_card_certs.ts

```diff
@@ -107,14 +127,19 @@ export async function parseCert(
  */
 export async function parseCardDetailsFromCert(
   cert: Buffer
-): Promise<CommonAccessCardDetails> {
-  const certDetails = await parseCert(cert);
+): Promise<CommonAccessCardDetails | undefined> {
+  try {
+    const certDetails = await parseCert(cert);
 
-  const { commonAccessCardId, givenName, middleName, familyName } = certDetails;
-  return {
-    commonAccessCardId,
-    givenName,
-    middleName,
-    familyName,
-  };
+    const { commonAccessCardId, givenName, middleName, familyName } =
+      certDetails;
+    return {
+      commonAccessCardId,
+      givenName,
+      middleName,
+      familyName,
+    };
+  } catch {
+    return undefined;
+  }
```

Comment:
Is this "bail on error" required? I'm guessing it's for when a card isn't configured. Maybe add a debug log in the `catch` block?

```ts
debug('error parsing CAC details from certificate: %s', error);
```

Reply:
yes, it's for the case where the card isn't configured yet and then on insert we read the data and boom.